assets/src/edit-story/app/story/utils/getStoryPropsToSave.js

```diff
@@ -31,6 +31,7 @@ function getStoryPropsToSave({ story, pages, metadata, flags }) {
     'featuredMedia',
     'password',
     'publisherLogo',
+    'storyPresets',
     'stylePresets',
```

Comment:
Should this one be renamed to `sitePresets` to say, that one attribute is for this story only and the other is for the entire site across all users? Or is it per-user, I actually can't remember?

Also, might I question the entire naming of "preset"? Maybe it's a bad name? Maybe it could be called "favorites", to better resemble how it's going to be used? And the global one might even be called something with "brand" or "theme". "Brand colors" and "brand typography" kind of works for me as the global ones. And then "story colors" for the local one. But not sure if it's _better_ 🤷 .

Feel free to give this a complete rethink when re-evaluating efforts - and remember that naming matters :)

Reply:
Changed the names to `currentStoryStyles` and `globalStoryStyles` as discussed. There are more places where to replace the word `preset`, however, will do this in another PR not to bloat this too much.